lib/src/js/actions/get_fee_history.js.dart

```diff
@@ -0,0 +1,36 @@
+part of '../wagmi.js.dart';
+
+/// [Documentation API](https://wagmi.sh/core/api/actions/GetFeeHistory)
+@JS()
+extension type JSGetFeeHistoryParameters._(JSObject _) implements JSObject {
+  external JSGetFeeHistoryParameters({
+    JSNumber? chainId,
+    JSNumber? blockCount,
```

Comment:
Following properties should be mandatory :
 - `blockCount`
 - `rewardPercentiles`